docs/arm2bicep.md

```diff
@@ -0,0 +1,12 @@
+# ARM Template syntax and Bicep equivalent syntax
+
+In Bicep, we have tried to make some common syntax in ARM templates that is quite verbose and change it to a terser equivalent. This doc is a simple table to look at all of the changes in one place.
+
+Scenario | ARM Template | Bicep
```

Comment:
Should we add a link to the relevant part of the spec for each example? (For example, the `&&` part could link to the list of operators.)

Should we include the following scenarios:
1. Generic expression guidance (`"[func()]"` vs `func()` in bicep)
1. `json('true')`, `json('false')`, `json('null')`

Should we also call out which aspects of the syntax behave exactly the same?

Reply:
yeah - all of these are good suggestions. I will update.

Reply:
actually for json('true') -> true(), those are now ARM template functions, so they can be used in either context, right?